### PR TITLE
Added support for previous an next item visibility

### PR DESCRIPTION
**Pull request by umasankar-yedida**
# Summary
 Added support for previous an next item visibility as mentioned in #37 

# Solution
Added additional 2 properties `pageWidth` and `paddingLeft` to make previous and next pages are visible

## Compatibility

| OS      | Implemented |
| ------- | :---------: |
| iOS     |    ❌     |
| Android |    ✅     |

## Checklist

- [X] I have tested this on a device and a simulator
- [ ] I added the documentation in `README.md`
- [ ] I mentioned this change in `CHANGELOG.md`
- [X] I updated the typed files (TS and Flow)
- [ ] I added a sample use of the API in the example project (`example/App.js`)

### CODE REVIEW

**Comment by ferrannp** [comment]
Thank you! I marked this on hold till 2.0 is on master and released.

**Comment by umasankar-yedida** [comment]
@ferrannp I'm started working on the changes. I'm new to iOS part so it is taking time. I will push changes as soon as possible. Thanks!!

**Comment by troZee** [comment]
for iOS you can check this implementation https://github.com/WenchaoD/FSPagerView 
and some solution here : https://stackoverflow.com/questions/20694679/horizontal-padding-between-uipageviewcontroller-child-view-controllers

File: js/ViewPagerAndroid.android.js [hunk 1]
```diff
@@ -123,6 +123,16 @@ type Props = $ReadOnly<{|
...
+
+  /**
+   * When it is value is 0.0 <= w <= 1.0, previous and next pages will be visible
+   */
+  pageWidth?: ?number,
+
+  /**
+   * This is necessary when pageWidth is specified. If not, UI looks ugly
+   */
+  paddingLeft?: ?number,
```
Comment (troZee) [inline js/ViewPagerAndroid.android.js]: How about padding right ?

File: example/ViewPagerAndroidExample.android.js [hunk 2]
```diff
@@ -179,6 +179,9 @@ export default class ViewPagerAndroidExample extends React.Component {
           onPageSelected={this.onPageSelected}
```
Comment (troZee) [inline example/ViewPagerAndroidExample.android.js]: I think, we don't want to split it into two files. `ViewPagerAndroidExample` should be for both platforms.

**Comment by troZee** [comment]
Hey, I have been investigation above issue. I was trying to apply some solution from stack overflow or invented by myself, but it did not bring anything good. Native View pager can render one page per time. It starts to load next one, while user swipes to go to next page. I see, that other people use another component (UICollectionView) to resolve this problem. eg https://github.com/WenchaoD/FSPagerView 

So as you can see this is known problem 
More to read:
https://stackoverflow.com/questions/42662386/uipageviewcontroller-padding-left-and-right
https://stackoverflow.com/questions/41284846/how-to-use-uipageviewcontroller-to-also-display-a-part-of-the-previous-and-next/41288766#41288766
https://stackoverflow.com/questions/15095375/preload-next-page-in-uipageviewcontroller
https://stackoverflow.com/questions/11659604/preloading-pages-of-uipageviewcontroller
https://stackoverflow.com/questions/36064524/uipageviewcontroller-with-preview-of-next-page
https://stackoverflow.com/questions/28807369/how-to-display-portion-of-next-page-in-uipageviewcontroller?rq=1

**Comment by troZee** [comment]
Hello 👋, this PR has been open for more than 2 months with no activity on it, hence I close it. In case of any questions, please let me know.

**Comment by liho00** [comment]
is this feature removed?

**Pull request closed without merging**